packages/at_client_mobile/lib/src/at_client_service.dart

```diff
@@ -267,61 +317,58 @@ class AtClientService {
   /// If atKeys are valid returns true; else, returns false.
   Future<bool> _validateAtKeys(String authenticateKey, String atSign,
       AtClientPreference atClientPreference) async {
-    AtLookupImpl atLookup = AtLookupImpl(
+    _atLookUp ??= AtLookupImpl(
         atSign, atClientPreference.rootDomain, atClientPreference.rootPort);
-    return await atLookup.authenticate(authenticateKey);
+    return await _atLookUp!.authenticate(authenticateKey);
```

Comment:
at_lookup.pkamAuthenticate is the correct method to use. atLookUp.authenticate(..) uses private key  the old way. I was not able to deprecate this method since dart analyzer was resulting in error in the calling packages

Reply:
Modified the code to use pkamAuthenticate method